save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/ResultsChecker.kt

```diff
@@ -26,7 +26,7 @@ class ResultsChecker(
      * @return [TestStatus]
      */
     @Suppress("TYPE_ALIAS")
-    internal fun checkResults(testFileName: String): TestStatus {
+    internal fun checkResults(testFileName: String): Triple<TestStatus, Int, Int> {
```

Comment:
I think `Pair<TestStatus, CountWarnings>` would be more readable

Reply:
Fixed